apps/cloud/src/api/user.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by acunningham on 22/05/17.
+ */
+import Store from '@raincatcher/store';
+import webapi, {ApiService, StoreApiService, WebApiConfig}  from '@raincatcher/webapi';
+import * as express from 'express';
+
+
+// Define User datatype
+interface User {
+  id: string,
+  username: string,
```

Comment:
In my opinion, username or email address would be better over a user ID. Its easier to know who the user is at first glance and LDAP/AD will nearly always have an email OR username field, but _maybe_ not a user ID.